sycl/test/functor/kernel_functor.cpp

```diff
@@ -18,27 +18,6 @@ constexpr auto sycl_read_write = cl::sycl::access::mode::read_write;
 constexpr auto sycl_global_buffer = cl::sycl::access::target::global_buffer;
 
```

Comment:
Not sure if the new testing guidelines apply to tests in sycl as well, but if it does, we should add a title comment here too.

Reply:
This one had a brief semi-hidden description here [L5](https://github.com/intel/llvm/blob/fe94fce04ebe790ab3536c72a3c004ff36e7f5dc/sycl/test/functor/kernel_functor.cpp#L5)
I however updated it.